engine/sbom/index.md

```diff
@@ -0,0 +1,91 @@
+keywords: Docker, sbon, Anchore, images, Syft, security
+toc_min: 1
+toc_max: 2
+---
+
+A Software Bill Of Materials (SBOM) is analogous to a packing list for a shipment; it’s all the components that make up the software, or were used to build it. For container images this includes the operating system packages that are installed (e.g.: ca-certificates) along with language specific packages that the software depends on (e.g.: log4j). The SBOM could include only some of this information or even more details, like the versions of components and where they came from.
+
+> :warning: *Note:* This command is considered _experimental_ and may change or be removed in the future.
+> Please leave feedback and report issues on [its repository](https://github.com/docker/sbom-cli-plugin).
+
+The experimental `docker sbom` command allows you to view the SBOM of a container image. Today, it does this today by scanning the layers of the image using the [Syft project](https://github.com/anchore/syft) but in future it may read the SBOM from the image itself or elsewhere.
+
+
+## Simple use
+
+To output a tabulated SBOM for an image simply run `docker sbom <image>:<tag>`:
+
+```console
+$ docker sbom neo4j:4.4.5
+Syft v0.43.0
+ ✔ Loaded image            
+ ✔ Parsed image            
+ ✔ Cataloged packages      [385 packages]
+
+NAME                      VERSION                        TYPE
+... 
+bsdutils                  1:2.36.1-8+deb11u1             deb
+ca-certificates           20210119                       deb
+...
+log4j-api                 2.17.1                         java-archive  
+log4j-core                2.17.1                         java-archive  
+...
+```
+
+Note that the output includes not only system packages but also software libraries used by applications in the container image.
+
+## Output formatting and saving outputs
+
+You can output the SBOM in standard formats like [SPDX](https://spdx.dev) and [CycloneDX](https://cyclonedx.org) along with the Syft and GitHub formats using the `--format` option.
+
+```console
+$ docker sbom --format spdx-json alpine:3.15
+{
+ "SPDXID": "SPDXRef-DOCUMENT",
+ "name": "alpine-3.15",
+ "spdxVersion": "SPDX-2.2",
+ "creationInfo": {
+  "created": "2022-04-06T21:13:32.035571Z",
+  "creators": [
+   "Organization: Anchore, Inc",
+   "Tool: syft-[not provided]"
+  ],
+  "licenseListVersion": "3.16"
+ },
+ "dataLicense": "CC0-1.0",
+ "documentNamespace": "https://anchore.com/syft/image/alpine-3.15-4b1b99d8-bbb5-4426-af8e-c510189134ab",
+ "packages": [
+  {
+   "SPDXID": "SPDXRef-1e3f3285636676f3",
+   "name": "alpine-baselayout",
+   "licenseConcluded": "GPL-2.0-only",
+   "description": "Alpine base dir structure and init scripts",
+   "downloadLocation": "https://git.alpinelinux.org/cgit/aports/tree/main/alpine-baselayout",
+   "externalRefs": [
+    {
+...
+}
+```
+
+You will notice that these outputs are more verbose and contain more information than the default tabulated output.
```

Comment:
Try avoiding future tense; this could be;

```suggestion
These outputs are more verbose and contain more information than the default tabulated output.
```